content/post/static-analysis-for-games.md

```diff
@@ -1,12 +1,101 @@
+**Focused tests** should be your first choice for this.
+They should be narrow, lightweight, and have clear success / failure criteria.
+In software, these would correspond to **unit tests**.
+
+When designing your focused tests, it can be helpful to:
+
+* Limit your scope to the elements that are core to the problem at hand
+* Create [**mock-ups**](TODO: add link) for an interlocking systems that can't be ignored
+* Use fast tools for making prototypes (pure design, pen-and-paper, board games, game making tools)
+* Ignore polish
+* Throw away your prototypes
+* Define clear [**user stories**](TODO: add link) for how the feature will be used
+* Design [**test cases**](TODO: add link) adversarially: try to develop diverse situations that maximize the likelihood that the system will fail. This helps catch potential issues and edge cases.
+* Write down clear success and failure criteria
+
+By narrowing the focus of your testing, you can save time on making prototypes, catch tricky edge cases, clearly define the problem you're trying to solve, and collectively decide whether a problem has been "fixed".
+
+## Classical playtesting
+
+In software, the classical approach to playtesting would be called **integration testing.**
+Testing your game from end-to-end, as if you were a player, can be great for catching unexpected problems, but comes with several very serious drawbacks:
+
+* Impossible to conduct without a reasonably complete prototype
+* Difficult to focus on a specific pieces of the game's design
+* Time consuming
+* New vs experienced player experience is very distinct
+* Players often do not accurately report subjective experience
+* Players often come not with problems found or observations, but with proposed solutions (which are often hard to disentangle)
+* Players are good at catching outwardly noticeable problems (such as confusion around a mechanic), but not finding deeper, systemic issues (such as poorly spent complexity)
+
+As a result, classical playtesting should often be considered as either a last resort, or as a "finishing pass" on a design you've already analyzed on it's own.
+This is particularly easy and effective if you have good tuning levers to work with, allowing your classical playtest to focus on nailing those values and improving game feel.
+
+## Conclusion
+
+In the end, classical playtesting can still be fun, informative, and highly effective, particularly for learning about subjective player experience, and performing holistic evaluation.
+But, as we hope we've convinced you in this blog post, it's only one tool in your toolbox.
+Use static analysis first to identify problems, hammer them out with focused tests and then use classical playtesting to refine feel and catch unforeseen issues.
+By doing this, you can catch mistakes early and guide your designs much more intentionally.
+As you sharpen these analysis skills, you'll eventually be able to accept, reject, and critique whole systems by inspection alone.
```

Comment:
Rereading, I really dislike this line. It's not a strong ending, and doesn't add anything interesting.

Reply:
I still really like it, but I'm more than willing to bend. Suggestions for alternates?

Reply:
Okay so the *actual* problem here is that "Conclusion" is, in general, a bad section. It's purposeless and just serves to rehash stuff people read just prior.

Instead, we need to be writing with a goal. In this case, I'm going to go with "Every tool has a place"